scene/resources/audio_stream_wav.h

```diff
@@ -33,6 +33,7 @@
 
 #include "servers/audio/audio_stream.h"
 
+#include "thirdparty/dr_libs/dr_wav.h"
```

Comment:
Having a thirdparty/dr_libs/godot.h file could enable us to import drwav inside a namespace also.
```cpp
namespace DrWav {
#include "thirdparty/dr_libs/dr_wav.h"
};
```

This would limit the global space pollution.

Reply:
Are you sure? This would result in a lot of `DrWav::drwav_*` declarations when they could be just `drwav_*`.

Reply:
I think this doesn't apply anymore after the rework.